packages/manager/modules/support/src/support.controller.js

```diff
@@ -0,0 +1,22 @@
+export default class {
+  /* @ngInject */
+  constructor(
+    $transitions,
+  ) {
+    this.$transitions = $transitions;
+  }
+
+  $onInit() {
+    const transitionCriteria = {
+      from: 'support.**',
+    };
+
+    this.$transitions.onStart(transitionCriteria, () => {
```

Comment:
to be clean you need to unregister the hook inside the $onDestroy method, otherwise if the controller is instantiated multiple times the hooks will be stacking up

Reply:
Excellent remark !